### ISSUE TITLE
In the recurring tasks, be able to change tasks' order 

### ISSUE THREAD

**Issue by rayon9coop** [coopcycle/coopcycle-web#3443]
**In order to** be more efficient :)

**As a** dispatcher

**I want** the possibility to change the tasks' order in the recurring tasks 

**Whereas currently** if you add one or several tasks to a recurring task, it can only go at the end, not in between two already existing tasks.

Thank you :)

**Comment by ERAMANrepartos** [comment]
@rayon9coop 
do you mean that
if i have a set of reoccurring tasks X,Y and Z that I have assigned to a rider. If I then try to assign a task W to a rider so that their task order is X,Q,Y,Z it does NOT allow me, and only allows task W to be ordered as W,X,Y,Z or X,Y,Z,W?

thanks for clarifying!

**Comment by rayon9coop** [comment]
No, this is not what I mean, sorry again for bad translation...

See for exemple when you create price rules, you can move un rule up or down by using your mouse. 

Here, in the creation of reccuring tasks's window, you can't move tasks up or down, so as a result, you can't change the order of tasks. 
For exemple you make this reccuring tasks 
PU1
D1
PU2
D2

After few weeks, your client ask you to doing PU2 and D2 before PU1 and D1 (forever, not just for one day). For that, i have to delete these recurring tasks and create new ones because I can't just move PU2 and D2 on top of PU1 and D1.

**Comment by ERAMANrepartos** [comment]
Got it! That makes a lot of sense thank
